build.sh

```diff
@@ -133,6 +133,13 @@ configure_user(){
     echo '%wheel ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/wheel-nopasswd
     # Contents of /etc/sudoers.d need not to be world writable
     chmod 600 /etc/sudoers.d/wheel-nopasswd
+
+    # Allow the builder user to run rootless podman
+    # Referenced at: https://github.com/containers/podman/issues/4056#issuecomment-1245715492
+    # Lifted from: https://github.com/containers/podman/blob/6e382d9ec2e6eb79a72537544341e496368b6c63/contrib/podmanimage/stable/Containerfile#L25-L26
+    echo -e "builder:1:999\nbuilder:1001:64535" > /etc/subuid
```

Comment:
This is fine, though ultimately I think what we want is to operate on `oci` directories for this - there's no reason to pull images into `containers-storage:` here.  I am not aware of tooling for this that lives in the github.com/containers ecosystem though.

There's https://github.com/opencontainers/umoci which I think handles this.

Reply:
Also worth noting that this is an overlapping change with https://github.com/coreos/coreos-assembler/pull/2985 which uses supermin to run `podman` instead there.

Ultimately what we want is https://kubernetes.io/docs/concepts/workloads/pods/user-namespaces/ stabilized - that unblocks clean nested containerization.

Reply:
> Also worth noting that this is an overlapping change with #2985 which uses supermin to run `podman` instead there.

Yep. My suggestion in https://github.com/coreos/coreos-assembler/pull/3096#issuecomment-1245892041 is that in the future (when we don't have clusters per architecture in RHCOS and just have multi-arch builders running FCOS like with do for the FCOS pipeline) we won't do that but rather run the builds through `cosa remote-build-container` which performs the builds via `podman --remote`. It's how we're building `cosa` and `fcos-buildroot` today.